src/app/campaign-summary.model.ts

```diff
@@ -1,5 +1,6 @@
 export type CampaignSummary = {
     percentRaised?: number|string;
+    isRegularGiving: boolean;
```

Comment:
this needs to be added in SF where we send these summaries. Doesn't actually have to be done before this is deployed to prod as long as there are no regular giving campaigns in prod since undefined is falsy.

Reply:
done